src/service.ts

```diff
@@ -12,7 +12,7 @@ import {
 
 export class Codecov {
   static baseUrl = "https://api.codecov.io";
-  static checkAuthPath = "/api/v2/github/codecov";
+  static checkAuthPath = "/api/v2/github/";
```

Comment:
https://docs.codecov.com/reference/root_list

This endpoint will work work just fine 🙂